nft_ingester/src/api/dapi/asset.rs

```diff
@@ -267,7 +269,8 @@ pub async fn get_by_ids<
                     let json_downloader = json_downloader.clone();
```

Comment:
What if instead of skipping the update we just make the whole update process as an asynchronous process in the background.

Reply:
That'll be a bigger story with the refresh queue. This one is a rather hot fix to deploy today and make live on Monday